From 4e7c6c8b67d6570418be06a399f684570491563d Mon Sep 17 00:00:00 2001
From: Jannick <thirdedition@gmx.net>
Date: Sat, 5 Jun 2021 19:25:58 +0200
Subject: [PATCH] m4 1.4.19: skip tests when temp directories cannot be
 created

* tests/init.sh:
  - skip test when required temporary directories cannot be generated
    due to access limitations.  Before test failed.
---
 tests/init.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/init.sh b/tests/init.sh
index 9ef8348..868fbdf 100644
--- a/tests/init.sh
+++ b/tests/init.sh
@@ -374,7 +374,7 @@ mktempd_ ()
     test $MAX_TRIES_ -le $i_ && break;
     i_=`expr $i_ + 1`
   done
-  fail_ "$err_"
+  skip_ "$err_"
 }

 # =============================================================================
@@ -404,7 +404,7 @@ setup_ ()
   # Create and enter the temporary directory.
   pfx_=`testdir_prefix_`
   test_dir_=`mktempd_ "$initial_cwd_" "$pfx_-$ME_.XXXX"` \
-    || fail_ "failed to create temporary directory in $initial_cwd_"
+    || skip_ "failed to create temporary directory in $initial_cwd_"
   cd "$test_dir_" || fail_ "failed to cd to temporary directory"
   # Set variables srcdir, builddir, for the convenience of the test.
   case $srcdir in
--
2.31.1.windows.1

